src/collections/_documentation/performance/distributed-tracing.md

```diff
@@ -12,105 +12,325 @@ Sentry's Performance features are currently in beta. For more details about acce
+
+- The browser's full process (_T_)
+  - XHR request to backend\* (_S_)
+  - Rendering confirmation screen (_S_)
+^
+- Your backend's processing of that request
+  - Function call to compute total (_S_)
+  - DB call to store order\* (_S_)
+  - API call to payment processor (_S_)
+  - Queuing of email confirmation\* (_S_)
+^
+- Your database's work updating the customer's order history (_T_)
+  - Individual SQL queries (_S_)
+^
+- The queued task of sending the email (_T_)
+  - Function call to populate email template (_S_)
+  - API call to email-sending service (_S_)
+  
+_Note:_ Starred spans represent spans that are the parent of a later transaction (and its root span).
+
+**Monitoring a Background Process**
+
+If your backend periodically polls for data from an external service, processes it, caches it, and then forwards it to an internal service, each instance of this happening is a trace, and you'd typically have transactions (_T_) and spans (_S_) for:
+
+- The cron job that completes the entire process (_T_)
+  - API call to external service (_S_)
+  - Processing function (_S_)
+  - Call to caching service\* (_S_)
+  - API call to internal service\* (_S_)
+^
+- The work done in your caching service (_T_)
+  - Checking cache for existing data (_S_)
+  - Storing new data in cache (_S_)
+^
+- Your internal service's processing of the request (_T_)
+  - Anything that service might do to handle the request (_S_)
+  
+_Note:_ Starred spans represent spans that are the parent of a later transaction (and its root span).
+  
+### Tracing Data Model
 
-Each [trace]({%- link _documentation/performance/performance-glossary.md -%}#trace) has a marker called a `trace_id`. Trace IDs are pseudorandom fixed-length alphanumeric character sequences.
+> "Show me your flowchart and conceal your tables, and I shall continue to be mystified. Show me your tables, and I won't usually need your flowchart; it'll be obvious." 
+>
+> -- [Fred Brooks](https://en.wikipedia.org/wiki/Fred_Brooks), The Mythical Man Month (1975)
 
-By collecting traces of your users as they use your applications, you can begin to reveal some insights such as:
+While the theory is interesting, ultimately any data structure is defined by the kind of data it contains, and relationships between data structures are defined by how links between them are recorded. Traces, transactions, and spans are no different.
 
-- What occurred for a specific error event, or issue
-- What conditions are causing bottlenecks or latency issues in the application
-- Which endpoints or operations consume the most time
+**Traces**
 
-### Trace Propagation Model
+Traces are not an entity in and of themselves. Rather, a trace is defined as the collection of all transactions that share a `trace_id` value. 
 
-Let's refer back to our earlier example of the web application consisting of these high-level components:
+**Transactions**
```

Comment:
Note: Once we have the docs written for the Transaction Payload in our develop docs we should cross link this. https://github.com/getsentry/develop/pull/25

Reply:
I'd be in favor of linking to this from the developer docs, so that people adding AM to SDKs know the API layer to build on top of their implementation, but I'm not sure we want to link to the payload schema here. That feels like an implementation detail, whereas this is more of a, "But what does it all _mean_?" kind of section.